server/gameconnection.py

```diff
@@ -294,14 +295,24 @@ async def handle_game_result(self, army, result):
         except (KeyError, ValueError):  # pragma: no cover
             self._logger.warning("Invalid result for %s reported: %s", army, result)
 
-    async def handle_operation_complete(self, army, secondary, delta):
-        if not int(army) == 1:
+    async def handle_operation_complete(
+        self,
+        primary_objectives_complete: typing.Any,
+        secondary_objectives_complete: typing.Any,
```

Comment:
These variable names seem a bit excessively long to me. Maybe we can call them `primary` and `secondary` and add a description of what they mean in a docstring? I know theres a point about self documenting code, but using 34% of available line length for 1 variable name seems pretty hefty to me:
```python3
>>> len("primary_objectives_complete")
27
>>> 27/79
0.34177215189873417

>>> len("secondary_objectives_complete")/79
0.3670886075949367
```